### PR TITLE
Requirements fix

### PR DESCRIPTION
**Pull request by GSokol**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
         }
     ],
     "require": {
-        "mockery/mockery": "~0.8"
+        "mockery/mockery": "~0.8",
+        "codeception/codeception": "^2.2"
```
Comment (enumag) [inline composer.json]: I think we can set this to "^2.0". The module should be compatible even with Codeception <2 but there is no need to support that now.

Comment (GSokol) [inline composer.json]: @enumag Done!

**Comment by enumag** [comment]
Looks good. Can you squash the commits?

**Comment by enumag** [comment]
Thanks. For the future you don't need to create a new PR, just use force-push.

**Pull request closed without merging**